modules/gridBidAdapter.js

```diff
@@ -427,6 +437,21 @@ function createBannerRequest(bid, mediaType) {
   return result;
 }
 
+function makeNewUserIdInFPDCookie() {
+  if (config.getConfig('cookieSettingAllowed')) {
+    const value = utils.generateUUID().replace(/-/g, '');
+    const expires = new Date(Date.now() + USER_ID_COOKIE_EXP).toISOString();
+
+    storage.setCookie(USER_ID_KEY, value, expires);
```

Comment:
why not use local storage?

Reply:
Expanding on this: cookies get appended to all requests that a browser makes to the publisher in the http headers, and can lead to overflows of CDN limits and failed requests. Eg cloudflare has a 4k limit. Publishers that make tons of requests to the same domain may see actual bandwidth consumed noticibly change. Local storage does not have this disadvantage.